### DISCUSSION TITLE
Customizing backspace

### DISCUSSION THREAD

**Discussion by foldingdemon** [AnySoftKeyboard/AnySoftKeyboard#2767] [Q&A]
Is there any way to customize the behavior of the backspace key to delete whole words instead of one letter at a time?

**Comment by noodlejetski** [comment]
Shift+Space deletes entire words.

**Reply by foldingdemon** [reply]
I have that enabled but it doesn't seem to work. I would prefer an option that allowed you to just use the backspace key by itself

**Reply by alchemiker** [reply]
You have to hold the shift button ⬆️ pressed and then to tap the backspace 🔙 while shift is still pressed.
What would be your suggestion to tell the backspace when to delete one sign and when to delete whole words?

**Reply by foldingdemon** [reply]
It could just be an option under tweaks. But functionally it could switch back to single char deletion if the backspace key was pressed twice within a certain amount of time. There was a keyboard that used to work like that on android, I think it was the Swype +dragon by Nuance.
